tests/visualization_tests/matplotlib_tests/test_parallel_coordinate.py

```diff
@@ -19,40 +23,98 @@ def test_target_is_none_and_study_is_multi_obj() -> None:
         plot_parallel_coordinate(study)
 
 
+def _fetch_objectives_from_figure(figure: Axes) -> List[float]:
+    # Fetch line plots in parallel coordinate.
+    line_collections = figure.findobj(LineCollection)
+    assert len(line_collections) == 1
+
+    # Fetch objective values from line plots.
+    objectives = [line[0, 1] for line in line_collections[0].get_segments()]
+    return objectives
+
+
+def _test_xtick_labels(axes: Axes, expected_labels: List[str]) -> None:
+    xtick_labels = axes[0].get_xticklabels()
+    for expected_label, xtick_label in zip(expected_labels, xtick_labels):
+        assert expected_label == xtick_label.get_text()
+
+
 def test_plot_parallel_coordinate() -> None:
 
     # Test with no trial.
     study = create_study()
     figure = plot_parallel_coordinate(study)
-    assert len(figure.get_lines()) == 0
+    assert len(figure.get_figure().axes) == 0 + 1
     plt.savefig(BytesIO())
 
     study = prepare_study_with_trials(with_c_d=False)
 
     # Test with a trial.
     figure = plot_parallel_coordinate(study)
-    assert len(figure.get_lines()) == 0
+    axes = figure.get_figure().axes
+    assert len(axes) == 3 + 1
+    # axes[0] is the objective vertical line.
+    assert axes[0].get_ylim() == (0.0, 1.0)
+    # axes[1] is colorbar.
+    assert axes[1].get_ylabel() == "Objective Value"
+    assert axes[1].get_ylim() == (0.0, 1.0)
+    # axes[2] is `param_a`'s vertical line.
+    assert axes[2].get_ylim() == (1.0, 2.5)
+    # axes[3] is `param_b`'s vertical line.
+    assert axes[3].get_ylim() == (1.0, 2.0)
+    objectives = _fetch_objectives_from_figure(figure)
+    assert objectives == [0.0, 1.0]
+    expected_labels = ["Objective Value", "param_a", "param_b"]
+    _test_xtick_labels(axes, expected_labels)
     plt.savefig(BytesIO())
 
     # Test with a trial to select parameter.
     figure = plot_parallel_coordinate(study, params=["param_a"])
-    assert len(figure.get_lines()) == 0
+    axes = figure.get_figure().axes
+    assert len(axes) == 2 + 1
+    assert axes[0].get_ylim() == (0.0, 1.0)
+    assert axes[1].get_ylabel() == "Objective Value"
+    assert axes[1].get_ylim() == (0.0, 1.0)
+    assert axes[2].get_ylim() == (1.0, 2.5)
+    objectives = _fetch_objectives_from_figure(figure)
+    assert objectives == [0.0, 1.0]
+    expected_labels = ["Objective Value", "param_a", "param_b"]
```

Comment:
In this case, `expected_labels` should be `["Objective Value", "param_a"]?

Reply:
You're right. Thanks!